### PR TITLE
Move more actions to v4 and remove autocomplete and user search from store

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Move more actions to v4 and remove autocomplete and user search from store.

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] All new/modified APIs include changes to the drivers

### CODE REVIEW

File: src/actions/general.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export function getClientConfig() {
 
 export function getLicenseConfig() {
     return bindClientFunc(
-        Client.getLicenseConfig,
+        Client4.getClientLicenseOld,
```
Comment (enahum) [inline src/actions/general.js]: what is with the old? API 4 returns something different?

File: src/actions/general.js [hunk 2]
```diff
@@ -94,8 +94,8 @@ export function setServerVersion(serverVersion) {
 
 export function setStoreFromLocalData(data) {
     return async (dispatch, getState) => {
-        Client.setToken(data.token);
-        Client.setUrl(data.url);
+        Client4.setToken(data.token);
+        Client4.setUrl(data.url);
```
Comment (enahum) [inline src/actions/general.js]: Im afraid that if we do this the `Client` won't have the token or the url set? can we keep it in both for now?

File: src/actions/general.js (hunk 1 above)
Comment (jwilander) [inline src/actions/general.js]: We return the client config and license as a map[string]string. I have a ticket in my backlog to fix it, hence why I'm doing this here

File: src/actions/general.js (hunk 2 above)
Comment (jwilander) [inline src/actions/general.js]: Yep, I can set both

**Pull request merged**